MantidPlot/make_package.rb.in

```diff
@@ -379,7 +399,7 @@ end
 # ---------------------------------------------
 # H5Py section
 # ---------------------------------------------
-h5py_path = "/usr/local/lib/python2.7/site-packages"
+h5py_path = homebrew_pip_site_packages
```

Comment:
Shouldn't this also be `pip_site_packages`? That will allow me to build the package too.

Reply:
I thought this always had to be homebrew to match the homebrew `hdf5` but yours is in the `pip` directory?

Reply:
I think I have `h5py` installed with brew, but it still lives in `/Library/Python/2.7/site-packages/h5py`. Is `h5py` our own formula? I could reinstall it, but I get:
```
brew info h5py
Error: h5py: Unsupported special dependency :python3
```

Reply:
It looks like it is https://github.com/mantidproject/homebrew-mantid/blob/master/h5py.rb. How strange that our is in `/usr/local`...

Reply:
Strange, I have `h5py` both in `pip list` and `brew list`, maybe I installed both...

Reply:
Yes, turns out I had both, removed the pip one. Now `import h5py` returns:
```
2): Library not loaded: /usr/local/opt/hdf5/lib/libhdf5.100.dylib
```
This does not exist, there is only `libhdf5.dylib` > libhdf5.101.dylib .
Can this mean there is a problem with the formula? or the version ?
Anyhow cannot reinstall the formula for this unsupported special dependency...

Reply:
That could be because `hdf5` was updated and `h5py` hasn't been. Perhaps your mantid formula repo needs updating? I think `python3` turned into `python` in later version of homebrew. Our `h5py` formula was updated but perhaps you don't have that?